libcxx/include/__configuration/abi.h

```diff
@@ -57,8 +57,6 @@
 // because it changes the mangling of the virtual function located in the vtable, which
 // changes how it gets signed.
 #  define _LIBCPP_ABI_BAD_FUNCTION_CALL_GOOD_WHAT_MESSAGE
```

Comment:
Let's add a benchmark. It looks like there are some interesting ones here: https://github.com/hiraditya/std-benchmark/blob/master/cxx/stringstream.bench.cpp

Let's also add the speedup results in the PR description.

Reply:
There's `locale/num_get.bench.cpp` already.